recipes/kmergenie/build.sh

```diff
@@ -14,8 +14,9 @@ export CPPFLAGS="-I${PREFIX}/include"
 
 mkdir -p $PREFIX/bin
 
+LC_ALL=C sed -i.bak 's' doobar
```

Comment:
Oh no this was just a terrible example ;)
You should probably do: `export LC_ALL=C`
And this is just a guess. Sorry for the confusing.